autoload/neomake/makers/ft/glsl.vim

```diff
@@ -0,0 +1,17 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#glsl#EnabledMakers() abort
+    return executable('glslc') ? ['glslc'] : []
+endfunction
```

Comment:
`executable` gets check automatically, so it could/should just return the default makers, i.e. `['glslc']`.